libs/cart/state/src/facades/cart/cart.facade.spec.ts

```diff
@@ -1618,4 +1607,100 @@ describe('DaffCartFacade', () => {
       expect(facade.canPlaceOrder$).toBeObservable(expected);
     });
   });
+
+  describe('getCartItemTotalDiscount', () => {
+
+    it('should be the cart item\'s discount total', () => {
+      const cart = cartFactory.create({
+        items: statefulCartItemFactory.createMany(2)
+      });
+      const expected = cold('a', { a: cart.items[0].total_discount });
+      facade.dispatch(new DaffCartLoadSuccess(cart));
+      expect(facade.getCartItemTotalDiscount(cart.items[0].item_id)).toBeObservable(expected);
+    });
+  });
+
+  describe('getCartItemDiscount', () => {
+
+    it('should be the cart item\'s discounted price', () => {
+      const cart = cartFactory.create({
+        items: statefulCartItemFactory.createMany(2)
+      });
+      const expected = cold('a', { a: daffDivide(cart.items[0].total_discount, cart.items[0].qty) });
```

Comment:
I think we should have an additional field on the cart item model for `discount` and do this computation once in the magento transformer. Hopefully other platforms will just provide this value too

Reply:
added